test/DynamoCoreWpfTests/ViewExtensions/PythonMigrationViewExtensionTests.cs

```diff
@@ -43,10 +33,6 @@ public void WillDisplayDialogWhenOpeningGraphWithIronPythonNodes()
         {
             DebugModes.LoadDebugModesStatusFromConfig(Path.Combine(GetTestDirectory(ExecutingDirectory), "DynamoCoreWpfTests", "python3DebugMode.config"));
             DynamoModel.IsTestMode = false;
-            // Arrange
-            RaiseLoadedEvent(this.View);
```

Comment:
This one I thought needed by some reason @reddyashish mentioned long time ago.. is the assumption no longer valid somehow?

Reply:
it's only required if we need whatever the Loaded event does to be re triggered - for this test it's not required.